scripts/modules/linear-label-management.js

```
@@ -0,0 +1,1262 @@
+			try {
+				return await operation();
+			} catch (error) {
+				lastError = error;
+
+				// Don't retry for certain error types
+				if (this._isNonRetryableError(error)) {
+					break;
+				}
+
+				if (attempt < this.config.maxRetries) {
+					const delay = this.config.retryDelay * Math.pow(2, attempt - 1);
```

Biome (1.9.4)

[error] Use the '**' operator instead of 'Math.pow'.

Unsafe fix: Use the '**' operator instead of 'Math.pow'.


(lint/style/useExponentiationOperator)